simple-stack-example-kotlin-masterdetail-fragments/src/main/java/com/example/ktdagger/userdetail/UsersAdapter.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.ktdagger.userdetail
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.ktdagger.MainActivity
+import com.example.ktdagger.R
+import com.example.ktdagger.realmobjects.UserRO
+import io.realm.Case
+import io.realm.Realm
+import io.realm.RealmChangeListener
+import io.realm.RealmResults
+import io.realm.Sort
+import io.realm.kotlin.where
+
+class UsersAdapter(private var realm: Realm, private var items: RealmResults<UserRO>?)
```

Comment:
I'd recommend using RealmRecyclerViewAdapter instead. (3.0.0) then you don't need to manually manage the change listener and would get diffs via OrderedRealmCollectionChangeSet

Reply:
I am not quite sure how to do this?